.github/resource/js/login.js

```diff
@@ -1,65 +1,57 @@
-
-// 비밀번호 input
-const password = document.querySelector('#password');
-
-// 링크
+const password = document.querySelector('.password');
 const hostname = '/folder';
+const pwd_icon = document.querySelectorAll('.pwd_icon');
+
+// 에러 메시지
+const ERROR_MESSAGE = {
+    email_empty: '이메일을 입력해 주세요.',
+    email_invalid: '올바른 이메일 주소가 아닙니다.',
+    email_check: '이메일을 확인해 주세요.',
+    email_use: '이미 사용 중인 이메일입니다.',
+    pwd_empty: '비밀번호를 입력해주세요.',
+    pwd_check: '비밀번호를 확인해 주세요.',
+    pwd_wrong: '비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.',
+    pwd_inc: '비밀번호가 일치하지 않아요.'
+}
 
-// 비밀번호 아이콘
-const pwd_icon = document.querySelector('.pwd_icon')
+// 에러 메시지 송출
+const showError = (targetEl, errorEl, errorType) => {
+    errorEl.style.display = 'block';
+    targetEl.classList.add('error');
+    errorEl.innerText = ERROR_MESSAGE[errorType];
+}
 
-email.addEventListener('focusout', function(e) {
-    if(email.value == '') {
-        error.style.display = 'block';
-        email.classList.add('error');
-        error.innerText = "이메일을 입력해 주세요";
-    } else if(!emailRegex.test(email.value)) {
-        error.style.display = 'block';
-        email.classList.add('error');
-        error.innerText = "올바른 이메일 주소가 아닙니다.";
-    } else if(emailRegex.test(email.value) && email.value !== '') {
-        error.style.display = 'none';
-        email.classList.remove('error');
-    }
-})
+// 에러 메시지 삭제
+const deleteError = (targetEl, errorEl) => {
+    targetEl.classList.remove('error');
+    errorEl.style.display = 'none';
+}
 
-password.addEventListener('focusout', function(e) {
-    if(password.value == '') {
-        error_pwd.style.display = 'block';
-        password.classList.add('error');
-        error_pwd.innerText = "비밀번호를 입력해주세요.";
-    } else if(email.value !== '') {
-        error_pwd.style.display = 'none';
-        password.classList.remove('error');
+// 이메일 값 비교
+const email_form = () => {
+    if(email.value === '') {
+        showError(email, error, 'email_empty');
+    } else if(!emailRegex.test(email.value)) {
+        showError(email, error, 'email_invalid');
     }
-})
+}
+email.addEventListener('focusout', email_form);
 
-form.onsubmit = function(e) {
-    if(email.value.trim() == 'test@codeit.com' && password.value == 'codeit101') {
-        e.preventDefault();
-        window.location.href = hostname;
+// 비밀번호 눈 아이콘 토글
+const pwd_check = (e) => {
+    e.target.classList.toggle('hide');
+    if(e.target.previousElementSibling.type === 'password') {
+        e.target.previousElementSibling.setAttribute('type', 'text');
     } else {
-        error.style.display = 'block';
-        email.classList.add('error');
-        error.innerText = "이메일을 확인해 주세요";
-        error_pwd.style.display = 'block';
-        password.classList.add('error');
-        error_pwd.innerText = "비밀번호를 확인해 주세요.";
+        e.target.previousElementSibling.setAttribute('type', 'password');
     }
 }
+pwd_icon.forEach((e) => {
+    e.addEventListener('click', pwd_check);
+});
```

Comment:
매개변수명도 명확하게 지어주셔야 합니다.
일반적으로 `e`는 이벤트 객체에 사용되기 때문에 헷갈릴 수 있겠네요.
위 리뷰를 참고해서 수정하면, 이런 식으로 작성할 수 있을 것 같습니다.
```suggestion
pwdIcons.forEach((pwdIcon) => {
    pwdIcon.addEventListener('click', pwd_check);
});
```